EESSI-install-software.sh

```diff
@@ -229,6 +229,27 @@ if [[ "${EESSI_CVMFS_REPO}" != /cvmfs/dev.eessi.io ]]; then
     ${TOPDIR}/install_scripts.sh --prefix ${EESSI_PREFIX}
 fi
 
+echo ">> Configuring EasyBuild..."
+
+# Make sure that we use the EESSI_CVMFS_INSTALL
+# Since the path is set when loading EESSI-extend, we reload it to make sure it works - even if it is already loaded
+# Note we need to do this after running install_cuda_and_libraries, since that does installations in the EESSI_SITE_INSTALL 
+unset EESSI_USER_INSTALL
+unset EESSI_PROJECT_INSTALL
+unset EESSI_SITE_INSTALL
+export EESSI_CVMFS_INSTALL=1
+module unload EESSI-extend
+
+# The EESSI-extend module is being loaded (or installed if it doesn't exist yet).
+# The script requires the EESSI_VERSION given as argument, a couple of
+# environment variables set (TMPDIR, EB and EASYBUILD_INSTALLPATH) and the
+# function check_exit_code defined.
+# NOTE, the script exits if those variables/functions are undefined.
+export EASYBUILD_INSTALLPATH=${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR_OVERRIDE}
```

Comment:
```suggestion

# Set a default installation path in case the EESSI-extend module itself needs to be installed
export EASYBUILD_INSTALLPATH=${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR_OVERRIDE}
```